gestalt/vault.py

```diff
@@ -192,20 +186,24 @@ def get(
         if "ttl" in requested_data:
             self._set_secrets_ttl(requested_data, key)
 
-        return returned_value_from_secret  # type: ignore
+        # repr is converting the string to RAW string since \\$ was returning $\
+        # then we are removing single quotes in the end to get final value
+        #
+        return str(repr(returned_value_from_secret)).replace("'", "")  # type: ignore
```

Comment:
what about secrets with a `'` in it?

Reply:
Good catch! I guess I'll just remove first and last char